test/pg-cdc-resumption/postgres-disable-select-permission.td

```diff
@@ -0,0 +1,11 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ postgres-execute connection=postgres://postgres:postgres@postgres
+REVOKE SELECT ON ALL TABLES IN SCHEMA "public" FROM materialize;
```

Comment:
Having one liners in a separate .td file is an anachronism but matches the rest of the style of the framework, which was created a long time ago.